tests/track/params_test.py

```diff
@@ -104,6 +105,12 @@ def test_generate_action_meta_data_without_id_conflicts(self):
                          next(params.GenerateActionMetaData("test_index", "test_type")))
 
     def test_generate_action_meta_data_with_id_conflicts(self):
+        def idx(id):
+            return "index", '{"index": {"_index": "test_index", "_type": "test_type", "_id": "%s"}}' % id
```

Comment:
Very minor comment, could use `.format()` for consistency going forwards.

Reply:
This is on the hot-code path where we decided against `#format()`.

Reply:
D'oh yes!